src/test/kotlin/day05/WandTest.kt

```
@@ -0,0 +1,95 @@
+    fun setUp() {
+    }
+
+    @After
+    fun tearDown() {
+    }
+
+    @Test
+    fun `wandConstructorNameIsNull`() {
+        try {
+            val wandAfterNameNull = Wand(null, 5.0)
+        } catch(e: IllegalArgumentException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)